docs/react/week-1/lesson.md

```diff
@@ -423,31 +407,30 @@ Or calculating new values:
-> 12. **(STRETCH GOAL)** Repeat the process with the `date` variable in the `CaughtPokemon.js` file
+| Exercise I (estimate: 10 min)                                                                                                                                                 |
+| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1. Using the `pokedex` React app that you created earlier, open the `App.js` file.                                                                                            |
+| 2. Pass a prop `appName="Pokedex"` to the `Logo` component.                                                                                                                   |
+| 3. Now open the `Logo.js` file.                                                                                                                                               |
+| 4. Delete the `appName` variable. What do you see in your web browser? Why?                                                                                                   |
+| 5. Change the `Logo` function to access the first argument and call it `props`. Use `console.log` to inspect the `props` variable.                                            |
+| 6. Change the usage of `appName` in the `<h1>` to be `props.appName` instead. Does this fix the problem? Why?                                                                 |
+| 7. Now open the `BestPokemon.js` file.                                                                                                                                        |
+| 8. Copy the `abilities` variable and then delete it from `BestPokemon.js`.                                                                                                    |
+| 9. Paste the `abilities` variable into `App.js`.                                                                                                                              |
+| 10. Pass the `abilities` variable as a prop to `BestPokemon` from `App.js`.                                                                                                   |
+| 11. In the `BestPokemon.js` file replace the existing usage of `abilities` with the `abilities` **prop**. You should still see the Pokemon ability names in your web browser. |
+| 12. **(STRETCH GOAL)** Repeat the process with the `date` variable in the `CaughtPokemon.js` file.                                                                            |
 
 ## Further reading
 
 Fed up of having to provide a prop for every component? Do you want to make your component use a value most of the time, but it can be overridden with a prop? This is a good time to use `defaultProps`. [This page on the React documentation](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) describes in more detail.
```

Comment:
Thanks, good point. I don't think I reviewed this when I was looking at the initial Hooks content. It is a bit frustrating that all of the official docs haven't been updated, but my understanding is that there is an on-going effort to update them.

I did some poking around and I discovered that there's a [proposal](https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md) to deprecate `defaultProps` on function components anyway (although I will note that was last year and I haven't heard any more about this since. They are supported in the latest version).

The proposal suggests moving to prop destructuring + default values, which I think is indeed too complex for it to be worth it.

So, I think I'm leaning towards removing this section altogether.